docs/introduction.md

```diff
@@ -2,9 +2,23 @@
 sidebar_position: 1
 ---
 # Why Unv?
-
-> I had learned many programming languages. This is my attempt to create a unique one for any platform.
->
-> _Kavindu Santhusa (Creater)_
-
-Currently Unv is in development.
+--
+We can't use one langauge for everything. Unv is a attempt to make a simple one for this purpose.With simple syntax similar to python , it can be used for any task.
+<br>

```

Comment:
```suggestion

```
In markdown empty+line breaks paragraphs, so `<br>` is unnecessary.
And this is a `mdx` file(a variant of markdown). so `<br>` is incorrect. `<br/>` is correct.